libs/ui-watt/src/lib/components/shell/nav-list/watt-nav-list-item.component.ts

```diff
@@ -14,34 +14,55 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
 import {
   ChangeDetectionStrategy,
   Component,
   Input,
   NgModule,
 } from '@angular/core';
-import { RouterModule } from '@angular/router';
-import { MatRippleModule } from '@angular/material/core';
+
+import { CommonModule } from '@angular/common';
 import { MatListModule } from '@angular/material/list';
+import { MatRippleModule } from '@angular/material/core';
+import { RouterModule } from '@angular/router';
 
 @Component({
   changeDetection: ChangeDetectionStrategy.OnPush,
   selector: 'watt-nav-list-item',
-  template: `<a
-    mat-list-item
-    mat-ripple
-    [routerLink]="link"
-    routerLinkActive="active"
-    ><ng-content></ng-content
-  ></a>`,
+  template: `
+    <a
+      *ngIf="isExternalLink; else internalLink"
+      mat-list-item
+      mat-ripple
+      [href]="link"
+      [attr.target]="target"
+      ><ng-container *ngTemplateOutlet="templateContent"></ng-container
+    ></a>
+
+    <ng-template #internalLink>
+      <a mat-list-item mat-ripple [routerLink]="link" routerLinkActive="active"
+        ><ng-container *ngTemplateOutlet="templateContent"></ng-container
+      ></a>
+    </ng-template>
+
+    <ng-template #templateContent>
+      <ng-content></ng-content>
+    </ng-template>
+  `,
 })
 export class WattNavListItemComponent {
   @Input() link: string | null = null;
+  @Input() target: '_self' | '_blank' | '_parent' | '_top' | null = null;
+
+  get isExternalLink(): boolean {
+    return /^(http|https)/i.test(this.link as string);
```

Comment:
suggestion: Instead of casting `null` as string, let's use the nullish coalescing operator to check the `link` value.

```suggestion
    return /^(http|https)/i.test(this.link ?? '');
```